tests/cypress/e2e/hedy_page/invalid_program.cy.js

```diff
@@ -0,0 +1,58 @@
+import {goToHedyLevel5Page, goToHedyPage} from "../tools/navigation/nav";
+
+describe('Error code gives correct error', () => {
+    it('Misspelled Keyword', () => {
+        const error_code = "prnt Hello world!"
+        const error_code2 = "print Hello world!\nak Hello world?"
+        const error_message = `prnt is not a Hedy level 1 command. Did you mean print?`;
+        const error_message2 = `ak is not a Hedy level 1 command. Did you mean ask?`;
+        goToHedyPage();
+
+        cy.get('#editor > .cm-editor > .cm-scroller > .cm-content').click();
+        cy.focused().type(error_code);
+
+        cy.get('#runit').click();
```

Comment:
Since the `#runit` button triggers a response from the server, I think it'd be wise to wrap it within an intercept, like we already do with the `#debug` button here: https://github.com/hedyorg/hedy/blob/179c72391ab0ddfa52cd6775fba9b9570d262155/tests/cypress/e2e/hedy_page/debugger.cy.js
It'd be something like this:

```suggestion
        cy.intercept('/parse').as('parse')
        cy.get('#runit').click();
        cy.wait('@parse')
```

What this will do is to wait for the server to respond in order to continue with the test. In the past we've had some flaky test due to not waiting properly.